### PR TITLE
Add odbc and jdbc 1.1.0.1 release artifact

### PR DESCRIPTION
**Pull request by joshuali925**
Signed-off-by: Joshua Li <joshuali925@gmail.com>

### Description
Add sql-odbc and sql-jdbc drivers to `_artifacts` for their 1.1.0.1 release
 
### Issues Resolved
linking #429 #431

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _artifacts/opensearch-drivers/_opensearch-jdbc-1.1.0.1.markdown [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+role: driver
+artifact_id: opensearch-sql-jdbc
+version: 1.1.0.1
+platform: all
```
Comment (stockholmux) [inline _artifacts/opensearch-drivers/_opensearch-jdbc-1.1.0.1.markdown:5]: `platform` should be `java`

File: _artifacts/opensearch-drivers/_opensearch-jdbc-1.1.0.1.markdown [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+architecture: all
```
Comment (stockholmux) [inline _artifacts/opensearch-drivers/_opensearch-jdbc-1.1.0.1.markdown:6]: `architecture` should be `jvm`

**Review by stockholmux — CHANGES_REQUESTED** [review]
A few little issues in the code review. In addition all the artifact filenames start with `_` which makes them hidden from rendering, so those will need to be removed (e.g. `_opensearch-jdbc-1.1.0.1.markdown ` -> `opensearch-jdbc-1.1.0.1.markdown`)

**Comment by joshuali925** [comment]
> A few little issues in the code review. In addition all the artifact filenames start with `_` which makes them hidden from rendering, so those will need to be removed (e.g. `_opensearch-jdbc-1.1.0.1.markdown ` -> `opensearch-jdbc-1.1.0.1.markdown`)

Thanks! updated

**Review by stockholmux — APPROVED** [review]
LGTM!

**Comment by stockholmux** [comment]
Fixes #429 , unblocks #431

**Comment by stockholmux** [comment]
@joshuali925 thanks for the contribution. It will go out on next deployment. Once I resolve #431, it will go on the downloads page.

**Pull request merged**